site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -55,11 +55,11 @@ public function openQueue() {
 
         $re = '/^[\sa-zA-Z0-9_\-]+$/m';
         preg_match_all($re, $queue_code, $matches_code, PREG_SET_ORDER, 0);
-        $matches_token = 1;
+        $matches_token = [];
```

Comment:
Do this for matches_code too

Reply:
Sure.  Matches code wasn't the one causing the crash, but better safe than sorry.